### DISCUSSION TITLE
（委托）委托方——仙舟联盟

### DISCUSSION THREAD

**Discussion by yunqiyuanshuai** [share121/inter-knot#21] [其他]
仙舟联盟立誓净除在星海中肆意荼毒诸界的丰饶孽物，他们需要你的帮助。
孽物不除，巡猎无已。 
                                                                ————华，云骑元帅

**Comment by yunqiyuanshuai** [comment]
仙舟翾翔，云骑常胜！

**Comment by ssdxs15** [comment]
这个委托我接了！

**Comment by ssdxs15** [comment]
委托我库库接

**Comment by YumeRin39Hana** [comment]
这委托就是法厄同来了也接不了，你告诉治安局的说不能换来一份防诈骗宣传单

**Comment by Landeeer** [comment]
我可以联系星穹列车

**Comment by MouxiuSama** [comment]
上面的莫要捣乱，此乃信息诈骗，这种玄幻的东西新艾利都怎么可能有，定然是电影看多了。说到电影，不如各位一起去支持下六分街那兄妹录像店？这种类型的片子他们那样一定有

**Comment by fulinshen** [comment]
这个委托我接了！
